lib/mayaUsd/ufe/UsdContextOps.cpp

```diff
@@ -324,59 +324,75 @@ class InsertChildAndSelectCommand : public Ufe::CompositeUndoableCommand
 //! \brief Undoable command for loading a USD prim.
 class LoadUnloadBaseUndoableCommand : public Ufe::UndoableCommand
 {
-public:
+protected:
+    LoadUnloadBaseUndoableCommand(const UsdPrim& prim, UsdLoadPolicy policy)
+        : _stage(prim.GetStage())
+        , _primPath(prim.GetPath())
+        , _policy(policy)
+    {
+    }
+
     LoadUnloadBaseUndoableCommand(const UsdPrim& prim)
         : _stage(prim.GetStage())
         , _primPath(prim.GetPath())
-        , _oldLoadRules(prim.GetStage()->GetLoadRules())
+        , _policy(getCurrentPolicy())
```

Comment:
Technically, this looks pretty dangerous because you call the function getCurrentPolicy from inside the constructor, and inside it uses _stage which may not be initialized at this point, depending on the order of initialization of data members.

Reply:
Yes, I knew it and ordered the members for it to work. I can easily fix this by putting the code in the body.